auth.py

```diff
@@ -0,0 +1,228 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from abc import ABCMeta, abstractmethod, abstractproperty
+from functools import wraps
+import hashlib
+import os.path
+import json
+import re
+
+from flask import request, Response
+
+WOTT_CREDENTIALS_PATH = '/opt/wott/credentials'
+WOTT_SCREENLY_CREDENTIAL_NAME = 'screenly_credentials'
```

Comment:
If it is used only to be appended with ".json" - I guess it could be named WOTT_SCREENLY_CREDENTIAL_FILE

Reply:
No, the logic implies that this name will be used in WoTT Credentials UI in the "Name" field.